tools/psoxy-test/cli-file-upload.js

```diff
@@ -48,6 +49,10 @@ const { version } = require('./package.json');
 
   program.parse(process.argv);
   const options = program.opts();
+  if (_.isString(options.deleteSanitizedFile)) {
+    options.deleteSanitizedFile = !(options.deleteSanitizedFile === 'false');
```

Comment:
I don't see this comparison, it will be always return same value as defined in `options.deleteSanitizedFile`

Reply:
Yes; hard to see, a mix of JS and `commander.js` weirdness.

`--delete-sanitized-file` is true by default even if absent as CLI option. So, if user wants to override, the option has to be `--delete-sanitized-file false`, and commander will parse the `false` as string. Then, `false` strict equality acts as type casting, and will get us back to the default `true`, if the option is any other value than `"false"`.

`--delete-sanitized-file false` -> `options.deleteSanitizedFile === false`
`--delete-sanitized-file foo` -> `options.deleteSanitizedFile === true`